commit 49e5784c73d05b25c6f69ed45ff11a1ec9bfb2c4 (refs/changes/60/2460/3)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2017-09-05T12:20:37-04:00 (2 years, 1 month ago)
    
    OS-6309 test-alias.js: add more verbose test messages
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/src/vm/tests/test-alias.js b/src/vm/tests/test-alias.js
index 88c6b2ee..46362028 100644
--- a/src/vm/tests/test-alias.js
+++ b/src/vm/tests/test-alias.js
@@ -1,4 +1,4 @@
-// Copyright 2015 Joyent, Inc.  All rights reserved.
+// Copyright 2017 Joyent, Inc.
 //
 // Tests for VM.lookup()
 //
@@ -81,7 +81,7 @@ function setAndCheckAlias(t, uuid, alias, base64, callback)
         }, function (cb) {
             // check that zonecfg value is correct
             getZonecfgAlias(uuid, function _checkZonecfgAlias(err, zalias) {
-                t.equal(zalias, base64);
+                t.equal(zalias, base64, 'alias the same in zonecfg');
                 if (err) {
                     t.ok(false, 'failed to get alias from zonecfg: '
                         + err.message);
@@ -98,7 +98,8 @@ function setAndCheckAlias(t, uuid, alias, base64, callback)
                     return;
                 }
 
-                t.equal((obj.alias == undefined) ? '' : obj.alias, alias);
+                t.equal((obj.alias == undefined) ? '' : obj.alias, alias,
+                    'alias the same from VM.load');
                 cb();
             });
         }
